pages/linux/edquota.md

```diff
@@ -0,0 +1,23 @@
+# edquota
+
+> Edit quota for a user or group.
+
+- Edit the user quota (default):
+
+`edquota --user {{user}}`
+
+- Edit the group quota:
+
+`edquota --group {{group}}`
+
+- Perform operations on given file system ONLY (default is to perform operations on all file systems with quota):
```

Comment:
I agree with @ostera. Also, it would be nice to clarify if this state is stored somewhere permanent, is only for the current session, only for the next command...